go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-knative/hack
 
-go 1.22.9
+go 1.24.0
```

Comment:
Are we driving default Golang version by inferring from hack's `go.mod`? In addition doesn't this min version break `go run/go install` we use everywhere to run various tools from o-k/hack?

Reply:
In cases where the infra is on 1.23 still.

Reply:
Let's use a common go setup then. And it needs bump to 1.23 as well. 

https://github.com/openshift-knative/hack/blob/ba4c9923390aec1ef4f47093af5bab8e8861bd4a/actions/setup-go/action.yaml

Reply:
Closing in favor of https://github.com/openshift-knative/hack/pull/610 then
/close